AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/TaskTests/ClientBehaviorTests/AuthenticationProviderDeleteUserTests.swift

```diff
@@ -75,16 +76,17 @@ class AuthenticationProviderDeleteUserTests: BasePluginTest {
     /// - Then:
     ///    - I should get a .service error with .network as underlying error
     ///
-    func testOfflineDeleteUser() async {
+    func testOfflineDeleteUser() async throws {
+        // TODO: How are client side retry errors now modeled?
```

Comment:
does this TODO need to be resolve?

Reply:
Thanks for catching this. Addressed in:
- https://github.com/aws-amplify/amplify-swift/pull/3287

Reply:
merged in [a76284a](https://github.com/aws-amplify/amplify-swift/pull/3248/commits/a76284a55880354a9c5a1b1f289fecf62155cb62)